testsuite/integration/secman/src/test/java/org/jboss/as/testsuite/integration/secman/ForwardJSPTestCase.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014, Red Hat, Inc., and individual contributors
```

Comment:
FYI it is now 2020

Reply:
Ah, yes. time flies :) 

Thank you for the review, updated to 2020. :)